test/07-openqa-label-known-issues.t

```diff
@@ -0,0 +1,109 @@
+
+try-client-output() {
+  client_output=''
+  try "$*"' && echo "$client_output"'
+}
+openqa-cli() {
+    local id=$(basename "$4")
+    cat "$dir/test/data/$id.json"
+}
+curl() {
+    if [[ "https://openqa.opensuse.org/tests/404/file/autoinst-log.txt" = "$7" ]] || [[ "https://openqa.opensuse.org/tests/414/file/autoinst-log.txt" = "$7" ]]; then
+        echo -n "404"
+    else
+        echo -n "200"
+    fi
+}
+comment_on_job() {
+    local id=$1 comment=$2 force_result=${3:-''}
+    echo "$comment"
+}
+
+try investigate_issue
+is "$rc" 1 'id required'
+# error in tap output is from here
+
+issues="159876
+[security] test fails in krb auto_review:\"com:/tmp/nfsdir /tmp/mntdir' failed\":force_result:softfailed
+openqa-force-result
+137420
+[qe-sap] test fails in network_peering auto_review:\"az network vnet peering create.+failed\"
+action
+73375
+Job incompletes with reason auto_review:\"(?m)api failure$\" (and no further details)
+action"
+
+# test data with reason but 404 in curl respond
+id=404
+testurl="https://openqa.opensuse.org/tests/${id}"
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing autoinst-log and with reason in job_data'
+has "$got" "seems like an old job without autoinst-log.txt" "old job without autoinst-log.txt"
+has "$got" "Job incompletes with reason" "investigation keep running without autoinst-log"
+
+# all requisite evidence are missing
+sed -i "s/api failure//" "test/data/${id}.json"
+tmplog=/tmp/autoinst-404.txt
+echo > $tmplog
+out=$tmplog
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing autoinst-log and unknown reason in job_data'
+has "$got" "does not have autoinst-log.txt or reason, cannot label" "investigation exits without reason and autoinst-log"
+sed -i "s/\"reason\": \"\"/\"reason\": \"api failure\"/" "test/data/${id}.json"
+
+#unknown reason
+sed -i "s/api failure/whatever/" "test/data/${id}.json"
+tmplog=/tmp/autoinst-404.txt
+echo "the reason is whatever" > $tmplog
+out=$tmplog
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing autoinst-log and unknown reason in job_data'
+has "$got" "Unknown test issue, to be reviewed" "investigation still label Unknown reason"
+sed -i "s/whatever/api failure/" "test/data/${id}.json"
+
+id=414
+testurl="https://openqa.opensuse.org/tests/${id}"
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with missing old autoinst-log and without reason in job_data'
+has "$got" "seems like an old job without autoinst-log.txt" "investigation reports old job"
+
+id=200
+testurl="https://openqa.opensuse.org/tests/${id}"
+tmplog=/tmp/autoinst-200.txt
+cp $autoinst_log $tmplog
+out=$tmplog
+try-client-output investigate_issue $id
+is "$rc" 0 'investigate_issue with autoinst-log and without reason'
+has "$got" "test fails in network_peering" "investigation label job with matched autoinst-log context"
+
+# handle_unreview branch
+echo >${tmplog}
```

Comment:
This also needs quoting and should use the style you used before for consistency:
```suggestion
echo > "$tmplog"
```